includes/class-wc-gutenberg-emails-loader.php

```diff
@@ -54,10 +54,41 @@ public function init_post_type() {
 	}
 
 	/**
-	 * Create emails content
+	 * Create email template posts.
 	 */
-	public function create_emails() {
-		// @todo
+	public function create_templates() {
+		global $wpdb, $pagenow;
+
+		// Only run this on the WooCommerce Emails page.
+		// phpcs:disable WordPress.Security.NonceVerification.NoNonceVerification
+		if ( 'edit.php' !== $pagenow || ! isset( $_GET['post_type'] ) || 'woocommerce_email' !== $_GET['post_type'] ) {
+			return;
+		}
+		// phpcs:enable
+
+		// Get already installed templates.
+		$installed_templates = $wpdb->get_col(
+			"SELECT post_name FROM {$wpdb->posts} WHERE post_type = 'woocommerce_email'"
+		);
+
+		$wc_emails = WC_Emails::instance();
+
+		foreach ( $wc_emails->emails as $key => $email ) {
+			if ( in_array( strtolower( $key ), $installed_templates, true ) ) {
+				continue;
+			}
+
+			wp_insert_post(
+				array(
+					'post_type'    => 'woocommerce_email',
+					'post_name'    => $key,
+					'post_title'   => $email->get_default_subject(),
+					'post_content' => '', // @todo Add blocks content for email.
+					'post_status'  => 'draft',
```

Comment:
Just curious here, but why creating them as drafts?

Reply:
Same question as above.  Figured these would only act as the email templates if they were published, otherwise default to PHP.

Let me know what you think.

Reply:
That makes sense, thanks for explaining!